lib/challenge.js

```diff
@@ -60,7 +76,11 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
-
+var Pet = function(name, species, breed, noise)
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.makeNoise = function(bark)
```

Comment:
This question wanted you to put the method on the prototype, not the instance.

Also, I'm noticing a recurring lack of curly braces following a function expression. Here is the **minimal function**, a function lacking everything but what is necessary to be syntactically correct: `function() {}`

Curly braces must occur even if the function has no statements to go inside them. They're non-negotiable. They're how the interpreter knows which statements are within the function, and if they're empty, they're how the interpreter knows there are no statements within the function. If they're absent, the interpreter concludes the code is broken and gives up.

Here's a sample solution to this question:

```js
var Pet = function Pet(name, species, breed, noise) {
    this.name = name;
    this.species = species;
    this.breed = breed;
    this.noise = noise;
}
Pet.prototype.makeNoise = function() {
    console.log(this.noise);
};
```

1/2